### PR TITLE
docs: fix broken links in profile readme

### LINKED ISSUE #353: Links in readme show 404 - not found on website
Keptn slack, meetings and contribute display 404-not found
Link to slack page is [https://keptn.sh/community/#slack](https://keptn.sh/community/#slack)

image
The community page also shows the same error.

### PR DESCRIPTION
**Pull request by AryanBakliwal**
Fixed broken links in the file [.github/profile/README.md](https://github.com/keptn/.github/blob/main/profile/README.md)

Fixes keptn/community#353

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 [![Website](https://img.shields.io/static/v1?label=Website&message=keptn.sh&color=blue)](https://keptn.sh/)
 [![Roadmap](https://img.shields.io/static/v1?label=Roadmap&message=public&color=green)](https://github.com/orgs/keptn/projects/1)
-[![Governance](https://img.shields.io/static/v1?label=Governance&message=process&color=yellow)](https://github.com/keptn/keptn/blob/master/GOVERNANCE.md)
-[![Participate](https://img.shields.io/static/v1?label=Contributing&message=guide&color=blue)](https://keptn.sh/community/)
-
+[![Governance](https://img.shields.io/static/v1?label=Governance&message=process&color=yellow)](https://github.com/keptn/community/tree/main/governance)
+[![Participate](https://img.shields.io/static/v1?label=Contributing&message=guide&color=blue)](https://keptn.sh/stable/docs/contribute/)
```
Comment (mowies) [inline profile/README.md]:
```suggestion
[![Participate](https://img.shields.io/static/v1?label=Contribution&message=guide&color=blue)](https://keptn.sh/stable/docs/contribute/)
```

File: profile/README.md [hunk 2]
```diff
@@ -15,19 +14,17 @@ For information about the history of the Keptn project, see the [Keptn Lifecycle
...
 There is a big community behind Keptn.
-We invite you to join us and participate. 
+We invite you to join us and participate.
 Connect to the community on
-[Keptn Slack](https://keptn.sh/community/#slack),
-join [our meetings](https://keptn.sh/community/#meetings),
-participate and [contribute](https://keptn.sh/community/contributing/).
-See also our 
-[Project Governance](https://github.com/keptn/keptn/blob/master/GOVERNANCE.md) and
+[Keptn Slack](https://cloud-native.slack.com/archives/C017GAX90GM),
```
Comment (mowies) [inline profile/README.md]:
```suggestion
[Keptn Slack](https://cloud-native.slack.com/messages/keptn/),
```

**Review by mowies — COMMENTED** [review]
2 small remarks, other than that LGTM

**Review by mowies — APPROVED** [review]

**Review by StackScribe — APPROVED** [review]

**Review by odubajDT — APPROVED** [review]

**Review by thisthat — APPROVED** [review]

**Comment by mowies** [comment]
thanks for the contribution @AryanBakliwal !

**Pull request merged**